bundle/Resources/config/services.yml

```diff
@@ -42,8 +42,37 @@ services:
         tags:
             - {name: twig.extension}
 
+    netgen.ezplatform_site.sort_clause_parser:
```

Comment:
`public: false` for all services.

Reply:
Added in https://github.com/netgen/ezplatform-site-api/pull/70/commits/96eea81268c31c3a91bcc77dd39bc9c6a712cbeb